account_bank_receipt/README.rst

```diff
@@ -0,0 +1,106 @@
+====================
+Account Bank Receipt
+====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge2| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/12.0-add-account_bank_receipt/account_bank_receipt
+    :alt: OCA/account-financial-tools
+.. |badge3| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-12-0-add-account_bank_receipt/account-financial-tools-12-0-add-account_bank_receipt-account_bank_receipt
+    :alt: Translate me on Weblate
+.. |badge4| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/92/12.0-add-account_bank_receipt
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| 
+
+This module allows you to easily manage transfer from intransit account the bank :
+you can select payment to many transfer.
+
+
+This module supports multi-currency.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+In the menu *Invoicing > Configuration > Accounting > Journals*, create a new
+journal:
+
+* Name: Bank Intransit
+* Type: Bank
+* Short Code: BIT (or any code you want)
+* Default Debit Account: select an account for received
```

Comment:
Not very clear on this. May be you can ask p'Lek to help what kind of account is needed and update this part again.